src/bsp/platform/stm32f4/xi_bsp_mem_stm32f4.c

```diff
@@ -7,8 +7,9 @@
 #include <xi_bsp_mem.h>
 #include <stdlib.h>
 #include <stdio.h>
-#include <FreeRTOS.h>
-#include <portable.h>
+
+void* pvPortMalloc( size_t );
+void vPortFree( void* );
 
```

Comment:
we don't generally use this prefix naming convention anywhere else in our codebase. 

Could you put a comment here saying that these are forward declarations of platform functions?